Marlin/configuration_store.cpp

```diff
@@ -187,20 +187,21 @@ void Config_StoreSettings()  {
   EEPROM_WRITE_VAR(i, planner.max_e_jerk);
   EEPROM_WRITE_VAR(i, home_offset);
 
+  uint8_t dummy_uint8 = 0;
   uint8_t mesh_num_x = 3;
   uint8_t mesh_num_y = 3;
   #if ENABLED(MESH_BED_LEVELING)
     // Compile time test that sizeof(mbl.z_values) is as expected
     typedef char c_assert[(sizeof(mbl.z_values) == (MESH_NUM_X_POINTS) * (MESH_NUM_Y_POINTS) * sizeof(dummy)) ? 1 : -1];
     mesh_num_x = MESH_NUM_X_POINTS;
     mesh_num_y = MESH_NUM_Y_POINTS;
-    EEPROM_WRITE_VAR(i, mbl.active);
+    dummy_uint8 = mbl.status & 0x01; // Do not save 'is active'
```

Comment:
Well, I was thinking of doing it in different ways. I wouldn't say naming "bits" and then require `_BV()` to be specially _good_ or safe. So I first was thinking of doing enums but with ready values, but felt that was doing more than necessary. So I ended up putting the real usage inside mbl methods, and really only this place being special, and its only a safe guard so the mesh isn't accidentally enabled, so it deserved a comment.